migrations/20210602_introductions.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS conn_intros (
+  intro_id BLOB NOT NULL PRIMARY KEY,
+  to_conn BLOB REFERENCES connections (conn_alias) ON DELETE SET NULL,
+  to_info BLOB, -- info about "to" connection sent to "re" connection
+  to_status TEXT, -- '', INTRO, INV, CON
+  re_conn BLOB REFERENCES connections (conn_alias) ON DELETE SET NULL,
+  re_info BLOB, -- info about "re" connection sent to "to" connection
+  re_status TEXT -- '', REQ, CON
+) WITHOUT ROWID;
+
+CREATE TABLE IF NOT EXISTS conn_invitations (
+  inv_id BLOB NOT NULL PRIMARY KEY,
+  via_conn BLOB REFERENCES connections (conn_alias) ON DELETE SET NULL,
+  ext_intro_id BLOB NOT NULL,
+  conn_info BLOB, -- info about another connection
+  conn_alias BLOB REFERENCES connections (conn_alias) -- created connection
```

Comment:
same - trying to figure out what is the real "unique" here.. is it just created connection `conn_alias`? can't it be made primary key?

Reply:
also please separate constraints below fields definition for consistency - we have it like this in the initial migration.
i.e.
```
PRIMARY KEY (inv_id),
FOREIGN KEY (conn_alias) REFERENCES ...
```

Reply:
This is a taste preference I disagree with. I prefer when constraints should be written on separate lines only when they are for multiple fields, I assumed that's what we had in initial schema.

Reply:
> what is the real "unique" here..

invitation ID that is referred to in commands

> is it just created connection conn_alias? can't it be made primary key?

no, because it is created in result of ACPT command - it is nullable (NULL initially)